File: python/day61.py
import re

# Check if string is 8 digits or 11 or starting with +, plus 12 digits (+63)

text = "+639674693577"

digits_only = re.sub(r'\D', '', text)
is_valid = len(digits_only) in (8, 11, 12)
print(is_valid)

File: js/day61.js
// Check if string is 8 digits or 11 or starting with +, plus 12 digits (+63)

const str = "+639674693578";

const digitsOnly = str.replace(/\D/g, '');
const isValid = digitsOnly.length === 8 || digitsOnly.length === 11 || digitsOnly.length === 12;
console.log(isValid);